### DISCUSSION TITLE
Compatibility ConBee II with ANgelEye / FireAngel Zigbee module P-Line

### DISCUSSION THREAD

**Discussion by LanMart** [zigpy/zigpy#910] [Q&A]
Hi All,

I hope I'm posting this on the right spot on Zigmee.
At Gamma I bought AngelEye/FireAngel smoke detectors and en CO-detector, all with a Zigbee module (P-line).

I'm using a Rasberry Pi with a ConBee stick and I cannot connect the smoke detector with my HomeAssistant.
Am I doing something wrong? Or is there a compatibility issue?
I was able to pair the smoke-detector with the ICS-2000 at my parent's, so it seems to work fine.

Suggestions are welcome

Martijn

**Comment by Hedda** [comment]
If compatibility/interoperability it probably belong as a "Device support request" in -> https://github.com/zigpy/zha-device-handlers/issues

Maybe ask to re-open this existing issue if it looks to be the same device -> https://github.com/zigpy/zha-device-handlers/issues/965

Note that you will probably need to provide much more detail about the exact identity of your specific device as well as debug logs.

Read reason why ZHA uses "quirk" device handler -> https://www.home-assistant.io/integrations/zha#zha-exception-and-deviation-handling

There is much more information about "ZHA Device Handlers" in -> https://github.com/zigpy/zha-device-handlers/blob/dev/README.md

PS: Also see ZHA general end-user support for and by the community here -> https://community.home-assistant.io/c/configuration/zha/48
